### PR TITLE
Remove old documents

### PR DESCRIPTION
**Pull request by kota2and3kan**
This PR removes the old document `how-to-use-customized-configuration.md`.
I moved each section in this document to the appropriate documents to organize documents.

Please take a look!

### CODE REVIEW

File: docs/how-to-use-customized-configuration.md [hunk 1]
```diff
@@ -1,294 +0,0 @@
-# How to use a customized properties file with Scalar Helm Charts
-
-You can set your customized properties files with Scalar Helm Charts (ScalarDB, ScalarDL Ledger, ScalarDL Auditor, and ScalarDL Schema Loader). Also, you can specify some flags in addition to the customized properties file with the ScalarDL Schema Loader chart. This document explains how to set properties files and flags with Scalar Helm Charts.
-
-## Set your properties file to a custom values file
```
Comment (kota2and3kan) [inline docs/how-to-use-customized-configuration.md:5]: I moved this section (how to set `*.properties`) to each document under [Configure a custom values file for Scalar Helm Charts](https://github.com/scalar-labs/helm-charts/blob/main/docs/configure-custom-values-file.md).

File: docs/how-to-use-customized-configuration.md [hunk 2]
```diff
@@ -1,294 +0,0 @@
...
-             scalar.db.contact_points=jdbc:postgresql://postgresql-ledger.default.svc.cluster.local:5432/postgres
-             scalar.db.username=postgres
-             scalar.db.password=postgres
-             scalar.db.storage=jdbc
-         ```
-
-1. Deploy Scalar products with the above custom values file.  
-   Please refer to the [Getting Started with Scalar Helm Charts](./getting-started-scalar-helm-charts.md) for more details on how to use each Helm Chart.
-
-## Use `Secret` resources to pass the credentials as the environment variables into the properties file
```
Comment (kota2and3kan) [inline docs/how-to-use-customized-configuration.md:61]: I moved this section to [How to use Secret resources to pass the credentials as the environment variables into the properties file](https://github.com/scalar-labs/helm-charts/blob/main/docs/use-secret-for-credentilas.md).

File: docs/how-to-use-customized-configuration.md [hunk 3]
```diff
@@ -1,294 +0,0 @@
...
-         ```properties
-         scalar.db.contact_points=jdbc:postgresql://postgresql-scalardb.default.svc.cluster.local:5432/postgres
-         scalar.db.username=postgres
-         scalar.db.password=postgres
-         scalar.db.storage=jdbc
-         ```
-
-   Please refer to the [Getting Started with Scalar Helm Charts](./getting-started-scalar-helm-charts.md) for more details on how to use each Helm Chart.
-
-## Mount arbitrary files like key and certificate files to the container in ScalarDL Helm Charts
```
Comment (kota2and3kan) [inline docs/how-to-use-customized-configuration.md:186]: I moved this section to [Mount key/certificate files to the pod in ScalarDL Helm Charts](https://github.com/scalar-labs/helm-charts/blob/main/docs/mount-key-and-cert-for-scalardl.md).

File: docs/how-to-use-customized-configuration.md [hunk 4]
```diff
@@ -1,294 +0,0 @@
...
-         ```console
-         $ ls -l /keys/
-         total 0
-         lrwxrwxrwx 1 root root 18 Jun 27 03:16 certificate -> ..data/certificate
-         lrwxrwxrwx 1 root root 18 Jun 27 03:16 private-key -> ..data/private-key
-         ```
-
-   Please refer to the [Getting Started with Scalar Helm Charts](./getting-started-scalar-helm-charts.md) for more details on how to use each Helm Chart.
-
-## Set flags with the ScalarDL Schema Loader chart
```
Comment (kota2and3kan) [inline docs/how-to-use-customized-configuration.md:272]: I moved this section to [Flags configurations (Optional based on your environment)](https://github.com/scalar-labs/helm-charts/blob/main/docs/configure-custom-values-scalardl-schema-loader.md#flags-configurations-optional-based-on-your-environment) in the [Configure a custom values file for ScalarDL Schema Loader](https://github.com/scalar-labs/helm-charts/blob/main/docs/configure-custom-values-scalardl-schema-loader.md).

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!

**Review by josh-wong — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**